src/components/contextual/pages/pool/PoolBalancesCard/BoostedPool.vue

```diff
@@ -35,10 +36,36 @@ const { explorerLinks } = useWeb3();
 function getUnderlyingTokens(address: string) {
   const linearPools = props.pool.onchain.linearPools;
 
+  if (linearPools == null) {
+    return [];
+  }
+
+  const tokenPriceAddress = linearPools[address].mainToken.address;
+
   return linearPools != null
-    ? [linearPools[address].mainToken, linearPools[address].wrappedToken]
+    ? [
+        linearPools[address].mainToken,
+        {
+          ...linearPools[address].wrappedToken,
+          tokenPriceAddress
```

Comment:
@garethfuller wasn't sure how to call it, tried to be descriptive here - the address of the token which will be used for pricing 👯